sdk/messaging/eventgrid/aznamespaces/CHANGELOG.md

```diff
@@ -4,10 +4,15 @@
 
 ### Features Added
 
-- Added in "binary mode" support, which can improve efficiency when sending a single event (PR#22582)
+- Supports API version `2024-06-01`, including support for release with delay.
+- Added in binary mode support, which can improve efficiency when sending a single event (PR#22582)
```

Comment:
Is binary mode still available or did that get removed in a previous commit?

Reply:
It's gone now, didn't mean to leave the changelog entry there.